### ISSUE TITLE
Incompatible with IE 11 (arrow functions in transpiled files)

### ISSUE THREAD

**Issue by EmilEriksen** [styled-icons/styled-icons#1278]
You can verify this by opening `node_modules/@styled-icons/styled-icon/dist/index.js`. It contains a few arrow functions. The transpiled icon files seem to be fine although I only had a quick glance, so it only seems to be the base icon that's problematic. Unfortunately, this completely breaks IE 11 compatibility for all packs. Would be awesome to have IE 11 compatibility :)

**Comment by jacobwgillespie** [comment]
Good catch, thanks for this report - the issue appears to be that all the modules correctly build ES5, except for `@styled-icons/styled-icon` which is mistakenly being built as ES6. I've opened #1279 to fix, after it passes the build/tests I'll push a new release.

**Comment by EmilEriksen** [comment]
Awesome! Thanks for the quick fix.

**Comment by jacobwgillespie** [comment]
This has been published as v10.5.0 - thanks again! 🎉

**Comment by Betree** [comment]
We still get reports on our bug tracker for a `Syntax error` on icons:

![image](https://user-images.githubusercontent.com/1556356/89986772-513a0100-dc7d-11ea-8bf2-0f58af2bed22.png)

<details>

```es6
import * as React from 'react';
import { StyledIconBase } from '@styled-icons/styled-icon';
export const Support = /*#__PURE__*/React.forwardRef((props, ref) => {
  const attrs = {
    "fill": "currentColor",
    "xmlns": "http://www.w3.org/2000/svg"
  };
  return /*#__PURE__*/React.createElement(StyledIconBase, Object.assign({
```

</details>

So far, it seems that the following libraries are impacted but there may be more:
- boxicons-regular
- fa-solid
- feather
- octicons

We didn't find a `10.5.0` version for these, we're currently using:

```es6
"@styled-icons/boxicons-regular": "10.14.1",
"@styled-icons/fa-brands": "10.14.1",
"@styled-icons/fa-solid": "10.14.1",
"@styled-icons/feather": "10.14.1",
"@styled-icons/icomoon": "10.14.1",
"@styled-icons/material": "10.14.1",
"@styled-icons/octicons": "10.14.1",
```

**Comment by jacobwgillespie** [comment]
@Betree thank you for catching this, a fix just released as 10.15.1. It looks like when we switched from TypeScript to Babel for compilation, the ESM module files were also ES6 (the CommonJS files have always been ES5). Now everything's ES5, so IE11-compatibility should be back. Let me know if you run into any issues.

**Comment by Betree** [comment]
Awesome, thank you for looking into this @jacobwgillespie!
